### PR TITLE
docs: migrate getting started pages

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
We are migrating getting started section of our documentation site

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [ ] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
 - [x] Request a review from Brad Paugh, David Becher, or Drew Chandler.

### CODE REVIEW

File: docs/getting-started/accessibility/fundamentals.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+As we consider the entire spectrum of possibilities, the benefits to the people we are designing for goes up dramatically. Accessible design is usable design, for more people in more circumstances.
+
+## Principles
+Building inclusive product experiences can be boiled down to the  **P.O.U.R.** principles:
+
+1.Perceivable.
+1.Operable.
+1.Understandable.
+1.Robust.
```
Comment (braddialpad) [inline docs/getting-started/accessibility/fundamentals.md]: something wrong with these

File: docs/getting-started/accessibility/fundamentals.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+**Examples:**
+
+- Predictable and consistent navigation.
+- Use of language such as abbreviations, jargon, reading levels.
+- Predictable and consistent actionable components.
+- Feedback and error handling.
+- Help, such as contextual help and error prevention.
+
+### Robus
```
Comment (braddialpad) [inline docs/getting-started/accessibility/fundamentals.md]: Robust

**Review by braddialpad — CHANGES_REQUESTED** [review]
Looks good, couple little things I noticed.

File: docs/getting-started/accessibility/fundamentals.md (hunk 1 above)
Comment (juliodialpad) [reply docs/getting-started/accessibility/fundamentals.md]: missing spaces, fixed

**Comment by juliodialpad** [comment]
> Looks good, couple little things I noticed.

good catch, thanks!

**Review by braddialpad — APPROVED** [review]

**Pull request merged**